### DISCUSSION TITLE
404 download page

### DISCUSSION THREAD

**Discussion by PleaseSmileToday** [KRTirtho/spotube#1407] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When clicking update from the app it gets a 404 page.
It goes to https://spotube.krtirtho.dev/other-downloads/stable-downloads

### Expected Behavior

It should be updated to a page with the actual download.

### Steps to reproduce

I opened the app, clicked on the download option and I also tried to do it from the settings.
And it opens a 404 error page.

### Operating System

Android

### Spotube version

3.5.0

### Installation source

Play Store (Android)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment, marked as answer]
The download link was changed to https://spotube.krtirtho.dev/downloads
